cmd/fluxd/main.go

```diff
@@ -178,8 +178,8 @@ func main() {
 		k8sNamespaceWhitelist = fs.StringSlice("k8s-namespace-whitelist", []string{}, "restrict the view of the cluster to the namespaces listed. All namespaces are included if this is not set")
 		k8sAllowNamespace     = fs.StringSlice("k8s-allow-namespace", []string{}, "restrict all operations to the provided namespaces")
 		k8sDefaultNamespace   = fs.String("k8s-default-namespace", "", "the namespace to use for resources where a namespace is not specified")
-
-		k8sVerbosity = fs.Int("k8s-verbosity", 0, "klog verbosity level")
+		k8sExcludeResource    = fs.StringSlice("k8s-exclude-resource", []string{"*metrics.k8s.io*", "*webhook.certmanager.k8s.io*"}, "do not include in Flux discovery the Kubernetes resources that match these glob expressions")
```

Comment:
Can you elaborate on the format to use?

Reply:
Also, the description may be confusing for the end user (since she doesn't know how discovery is used internally).

Reply:
```suggestion
		k8sExcludeResources    = fs.StringSlice("k8s-exclude-resource", []string{"*metrics.k8s.io*", "*webhook.certmanager.k8s.io*"}, "do not attempt to obtain cluster resources whose group/version that matches these glob expressions")
```